src/AppBundle/Entity/Employee.php

```diff
@@ -224,4 +225,27 @@ public function getRoles()
     {
         return $this->roles;
     }
+
+    /**
+     * Get deletedAt
+     *
+     * @return \DateTime
+     */
+    public function getDeletedAt()
```

Comment:
https://github.com/geekhub-php/CheTheatre/commit/5de95ef4d10641f7b4597d03984d24e3b29d79e5
done in this commit. in the final version in branch they are in trait